File: php-slim/lib/Model/RequestTimeFilterFastArrivalManyToOneSearch.php
<?php
/**
 * RequestTimeFilterFastArrivalManyToOneSearch
 */
namespace OpenAPIServer\Model;

/**
 * RequestTimeFilterFastArrivalManyToOneSearch
 */
class RequestTimeFilterFastArrivalManyToOneSearch
{
    
    /** @var string $id */
    private $id;
    
    /** @var string $arrivalLocationId */
    private $arrivalLocationId;
    
    /** @var string[] $departureLocationIds */
    private $departureLocationIds;
    
    /** @var \OpenAPIServer\Model\RequestTransportationFast $transportation */
    private $transportation;
    
    /** @var int $travelTime */
    private $travelTime;
    
    /** @var \OpenAPIServer\Model\RequestArrivalTimePeriod $arrivalTimePeriod */
    private $arrivalTimePeriod;
    
    /** @var \OpenAPIServer\Model\RequestTimeFilterFastProperty[] $properties */
    private $properties;
}

File: fsharp-giraffe-server/traveltimeplatform/src/model/RequestTimeFilterFastArrivalManyToOneSearch.fs
namespace traveltimeplatform.Model

open System
open System.Collections.Generic
open traveltimeplatform.Model.RequestArrivalTimePeriod
open traveltimeplatform.Model.RequestTimeFilterFastProperty
open traveltimeplatform.Model.RequestTransportationFast

module RequestTimeFilterFastArrivalManyToOneSearch = 

  //#region RequestTimeFilterFastArrivalManyToOneSearch


  type RequestTimeFilterFastArrivalManyToOneSearch = {
    Id : string;
    ArrivalLocationId : string;
    DepartureLocationIds : string[];
    Transportation : RequestTransportationFast;
    TravelTime : int;
    ArrivalTimePeriod : RequestArrivalTimePeriod;
    Properties : RequestTimeFilterFastProperty[];
  }
  //#endregion
  